<?xml version="1.0"?>
<launch>
	<param name="service_name" value="/server_led_com"/>
	<param name="path_device" value="/dev/ttyUSB_LEDS"/>
	
	<param name="topic_direction" value="/rb1_base_a/robotnik_base_control/cmd_vel"/>
	<param name="topic_elevator" value="/rb1_base_a/robotnik_base_control/elevator_status"/>
	
	<param name="max_value_lineal_x" value="0.5"/>
	<param name="max_value_lineal_y" value="0.5"/>
	<param name="max_value_angular_z" value="0.5"/>
	<param name="porc_min_linear_x" value="0.01"/>
	<param name="porc_min_linear_y" value="0.03"/>
	<param name="porc_min_angular_z" value="0.1"/>
	<param name="leds" value="120"/>
	
	<param name="led_start_front" value="16"/>
	<param name="led_end_front" value="44"/>
	
	<param name="led_start_right" value="45"/>
	<param name="led_end_right" value="75"/>
	
	<param name="led_start_back" value="76"/>
	<param name="led_end_back" value="104"/>
	
	<param name="led_start_left" value="105"/>
	<param name="led_end_left" value="15"/>
	
	<param name="led_start_front_right" value="0"/>
	<param name="led_end_front_right" value="16"/>
	
	<param name="led_start_front_left" value=""/>
	<param name="led_end_front_left" value=""/>
	
	<param name="led_start_back_left" value=""/>
	<param name="led_end_back_left" value=""/>
	
	<param name="led_start_back_right" value=""/>
	<param name="led_end_back_right" value=""/>
	<node name="robotnik_leds_service" pkg="robotnik_leds" type="robotnik_leds_service" respawn="false" output="screen"/> 
    <node name="robotnik_leds_node" pkg="robotnik_leds" type="robotnik_leds_node" respawn="false" output="screen"/>        
</launch>
